src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -12,20 +12,24 @@
             cursor: pointer;
         }
 
-        .signpost-action {
-            min-width: $clr_baselineRem_1;
-            margin: 0;
-            padding: 0;
+        .signpost-trigger {
             color: $clr-dark-midtone-gray;
-            outline: none;
+            border: none;
 
-            clr-icon {
-                width: $clr_baselineRem_1;
-                height: $clr_baselineRem_1;
+            &:active {
+                border: none;
+            }
+
+            &.active, &:hover {
+                color: $clr-action-blue;
+                background-color: transparent;
             }
 
-            &:hover, &.active {
+            &:focus {
                 color: $clr-action-blue;
+                outline: 0;
+                border: 1px solid $clr-action-blue;
+                border-radius: $clr-default-borderradius;
```

Comment:
We have `tabindex=0` now on the trigger directive so we will have a default browser focus there. We don't need to now manipulate that default browser focus, right? So I think we don't need all these border and outline styles.

Reply:
Yeah, this is still in progress. I'm working through a slightly different issue. But, this code will be removed as you indicated. Thanks.